src/linker/Linker/Driver.cs

```diff
@@ -434,6 +434,10 @@ protected int SetupContext (ILogger customLogger = null)
 
 						continue;
 
+					case "--generate-warning-suppressions":
```

Comment:
This needs documentation what is this suppose to produce. Right now it outputs a list of suppression of unsuppressed warnings. Is it intentional that it does not include any global suppression?